src/lib/Components/Inbox/ActiveBids/ActiveBid.tsx

```diff
@@ -113,21 +113,23 @@ class ActiveBid extends React.Component<RelayProps, State> {
   }
 
   render() {
-    const bid = this.props.bid.most_recent_bid
-    const imageURL = bid.sale_artwork.artwork.image.url
-    const lotNumber = bid.sale_artwork.lot_number
-    const artistName = bid.sale_artwork.artwork.artist_names
-
-    const headline = `Lot ${lotNumber} · ${artistName} `
-
-    const isInOpenLiveAuction = this.props.bid.sale && this.props.bid.sale.is_live_open
-    const subtitle = isInOpenLiveAuction ? "Live bidding now open" : `Current Bid: ${bid.max_bid.display} `
+    const {
+      bid: {
+        sale: { is_live_open },
+        most_recent_bid: { max_bid, sale_artwork: { artwork: { image, artist_names }, highest_bid, lot_number } },
```

Comment:
Wanted to make the . . . a little more readable but damn, prettier fail due to 120 print width, reverting...

I _really_ wish prettier allowed for nested destructuring: https://github.com/prettier/prettier/issues/2550